pkg/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -20,9 +20,13 @@ import (
 )
 
 const (
-	annotationUnprivileged      = "container.apparmor.security.beta.kubernetes.io/dynatrace-oneagent"
+	annotationUnprivileged     = "container.apparmor.security.beta.kubernetes.io/dynatrace-oneagent"
+	annotationTenantTokenHash  = api.InternalFlagPrefix + "tenant-token-hash"
+	annotationEnableDsEviction = "cluster-autoscaler.kubernetes.io/enable-ds-eviction"
+
 	annotationUnprivilegedValue = "unconfined"
-	annotationTenantTokenHash   = api.InternalFlagPrefix + "tenant-token-hash"
+	annotationTrueValue         = "true"
```

Comment:
```suggestion
	StringTrueValue         = "true"
```
IMO it's nothing about annotations, yes we use it there only, but it's just confusing to me

Reply:
cc @0sewa0 maybe it's better to fix separately, I did a quick search and we have all combinations:
- tokenSecretHashAnnotation
- AnnotationXXX
- justWithoutAnnotation

https://github.com/dynatrace/dynatrace-operator/blob/8c0742c8e8bfd3b25c5eb2bb004ef8f71b387fc6/pkg/controllers/dynakube/kspm/daemonset/volumes.go#L13

Reply:
(I always found putting `"true"` in a const to be a bit of an overkill :) )

Reply:
yes, i did search in k8s/api codebase - they never do it.

so I'm thinking let's be consistent, since we are not changing entire codebase just few files, let's revert back to "true".

Reply:
I will remove the change then :+1: thanks for the input guys

Reply:
Fixed https://github.com/Dynatrace/dynatrace-operator/pull/4869/commits/3e0096a920b7a5e2f47cf044009feab2e222c54e